Gopkg.toml

```diff
@@ -0,0 +1,83 @@
+# Gopkg.toml example
```

Comment:
Let's remove this commented out portion and we should be good to go!

Reply:
Done :)